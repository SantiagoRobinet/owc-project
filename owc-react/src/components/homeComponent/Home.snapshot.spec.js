import renderer from 'react-test-renderer';
import React from 'react';
import HomeComponent from './Home';

describe('Home snapshot', () => {
    const props = {
        label: 'click me'
    }
    const homeTree = renderer.create(
        <HomeComponent {...props} />
    );
    it('should match', () => {
        expect(homeTree.toJSON()).toMatchSnapshot();
    });

});