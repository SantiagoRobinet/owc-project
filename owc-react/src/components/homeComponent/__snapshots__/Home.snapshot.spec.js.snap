// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot should match 1`] = `
<div
  className="button__container"
>
  <owc-button
    label="click me"
  />
</div>
`;
